src/enums/team.js

```diff
@@ -69,6 +69,11 @@ export default {
       title: 'Icahn School of Medicine at Mount Sinai',
       link: 'https://www.linkedin.com/in/anna-blech-121277138',
     },
+    {
+      name: 'Anya Lehr',
+      title: 'Ex-Vice Media Director of Operations',
```

Comment:
nit: We seem to mix "Ex-" and "Former".  "Former" sounds a little better to me, but I don't have a strong preference.  Anyway, approving as-is, but maybe something to clean up the next time we're making edits.